keyboards/ergodox_ez/util/keymap_beautifier.py

```diff
@@ -0,0 +1,154 @@
+    output += beautify_keys_section("\n".join(src["keys"])) + "\n"
+    output += "\n".join(src["after"]) + "\n"
+    return output
+
+def beautify_keys_section(src):
+    #parsed = pycparser.parse_file(c_filename)
+    parsed = pycparser.CParser().parse(src)
+    layer_output = []
+    for layer in parsed.children()[0][1].init.exprs:
+        ascii_key_symbols = transform_layer(layer)
+        layer_output.append(pretty_output_layer(layer.name[0].value, ascii_key_symbols))
+    return """
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {{
+/*
+ * ,--------------------------------------------------.    ,--------------------------------------------------.
+ * |    0   |   1  |   2  |   3  |   4  |   5  |  6   |    |  38  |  39  |  40  |  41  |  42  |  43  |   44   |
+ * |--------+------+------+------+------+------+------|    |------+------+------+------+------+------+--------|
+ * |    7   |   8  |   9  |  10  |  11  |  12  |  13  |    |  45  |  46  |  47  |  48  |  49  |  50  |   51   |
+ * |--------+------+------+------+------+------|      |    |      |------+------+------+------+------+--------|
+ * |   14   |  15  |  16  |  17  |  18  |  19  |------|    |------|  52  |  53  |  54  |  55  |  56  |   57   |
+ * |--------+------+------+------+------+------|  26  |    |  58  |------+------+------+------+------+--------|
+ * |   20   |  21  |  22  |  23  |  24  |  25  |      |    |      |  59  |  60  |  61  |  62  |  63  |   64   |
+ * `--------+------+------+------+------+-------------'    `-------------+------+------+------+------+--------'
+ *   |  27  |  28  |  29  |  30  |  31  |                                |  65  |  66  |  67  |  68  |  69  |
+ *   `----------------------------------'                                `----------------------------------'
+ *                                       ,-------------.  ,-------------.
+ *                                       |  32  |  33  |  |  70  |  71  |
+ *                                ,------+------+------|  |------+------+------.
+ *                                |      |      |  34  |  |  72  |      |      |
+ *                                |  35  |  36  |------|  |------|  74  |  75  |
+ *                                |      |      |  37  |  |  73  |      |      |
+ *                                `--------------------'  `--------------------'
+ */
+{}
+
+}};
+    """.format(",\n\n".join(layer_output),)
+
+def unpack_function(f):
+    name = f.name.name
+    args = []
+    for arg in f.args.exprs:
+        if type(arg) is pycparser.c_ast.Constant:
+            args.append(arg.value)
+        elif type(arg) is pycparser.c_ast.ID:
+            args.append(arg.name)
+    return "{}({})".format(name, ",".join(args))
+
+def transform_raw_key_symbol(raw):
+    if type(raw) is pycparser.c_ast.ID:
+        if raw.name in ["KC_TRNS", "KC_TRANSPARENT"]:
+            return "_______"
+        return raw.name
+    elif type(raw) is pycparser.c_ast.FuncCall:
```

Comment:
```suggestion
    if type(raw) is pycparser.c_ast.FuncCall:
```
Don't need `elif` after unconditional `return`.